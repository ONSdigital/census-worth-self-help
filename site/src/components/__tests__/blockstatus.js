import React from "react"
import renderer from "react-test-renderer"
import BlockStatus from "../blockstatus"

describe("BlockStatus", () => {
  it("renders correctly", () => {
    const tree = renderer
      .create(<BlockStatus title="title" subtitle="anything" />)
      .toJSON()
    expect(tree).toMatchSnapshot()
  })
})