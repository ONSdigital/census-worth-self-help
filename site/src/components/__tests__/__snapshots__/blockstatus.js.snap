// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BlockStatus renders correctly 1`] = `
<div
  css={
    Object {
      "map": undefined,
      "name": "bn78dc",
      "next": undefined,
      "styles": "
        box-shadow: 0 2px 4px 0 rgba(212, 212, 212, 0.5);
        border: solid 1px #ececed;
        background-color: rgb(255, 255, 255);
        height: 74px;
        display: flex;
        align-items: center;
        justify-content: center;
      ",
    }
  }
>
  <div
    css={
      Object {
        "map": undefined,
        "name": "1w87xe6",
        "next": undefined,
        "styles": "
          color: rgb(0,61,89);
          
    display: flex;
    justify-content: center;
    flex-direction: column;
  
          font-size: 30px;
          flex: 1 1 0;
          align-items: center;
          padding: 0px 5px;
        ",
      }
    }
  />
  <div
    css={
      Object {
        "map": undefined,
        "name": "sj34c4",
        "next": undefined,
        "styles": "
          
    display: flex;
    justify-content: center;
    flex-direction: column;
  
          text-align: center;
        ",
      }
    }
  >
    <div
      className="Card-heading-Style"
      css={
        Object {
          "map": undefined,
          "name": "2e2jvx",
          "next": undefined,
          "styles": "
            margin-bottom: 5px;
          ",
        }
      }
    >
      title
    </div>
    <div
      className="Card-sub-head-Style-gray"
    >
      anything
    </div>
  </div>
  <div
    css={
      Object {
        "map": undefined,
        "name": "531ogy",
        "next": undefined,
        "styles": "
          flex: 1 1 0;
        ",
      }
    }
  />
</div>
`;
